Products/CMFPlomino/PlominoForm.py

```diff
@@ -1335,6 +1335,37 @@ def validation_errors(self, REQUEST):
                     errors=json.dumps({'success': True}))
 
 
+    security.declarePrivate('_get_js_hidden_subforms')
+    def _get_js_hidden_subforms(self, REQUEST, doc, validation_mode=False):
+        hidden_forms = []
+        hidewhens = json.loads(
```

Comment:
By the way, I see we do a similar thing in `getHidewhenAsJSON` at the moment, so we can clean that at the same time.